app/migrations/versions/2c4716931a43_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 2c4716931a43
+Revises: 7c0e719eccd9
+Create Date: 2019-10-18 20:49:39.227852
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2c4716931a43'
+down_revision = '7c0e719eccd9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('user', sa.Column('business_usage_id', sa.Integer(), nullable=True))
```

Comment:
uh oh i added the same thing in my PR, we're going to have collisions :|

Reply:
oh wait i made it a custom attribute :D so we're good. any reason for it to be in user vs to be a custom attribute?

Reply:
Yea, i thought this would be a common attributes so most user would have a business_usage_id and i thought custom attributes is more for the uncommon attributes that user can have. 

I also think it may make querying for business_usage_id faster. May also come in handy if we want to implment Wills solution of the (~4+ layers deep) tranfer usage. 

What do you think?

Reply:
I _think_ this makes slightly more sense as an actual column rather than a custom attribute; while my natural inclination is to keep the user table as constrained to key information rather than supplementary data, extending on @RubenGeo 's point, given that business_id is a foreign key, it probably makes more sense to specify it as such.*

Actually on that note, the column definition above doesn't have it as a foreign key - perhaps this should be updated if we do keep it as a column?

Reply:
Good point, i updated this to a foreign key :+1: